Python/ceval_gil.c

```diff
@@ -278,6 +278,10 @@ static void recreate_gil(struct _gil_runtime_state *gil)
 static void
 drop_gil(struct _ceval_state *ceval, PyThreadState *tstate)
 {
+    /* We shouldn't be using a thread state that isn't viable any more. */
```

Comment:
This comment is cryptic here. It doesn't say why it's here nor in which situation the "non-viable thread state" occurs.

Reply:
I've clarified the comment.